conda-env/ci.yml

```diff
@@ -0,0 +1,30 @@
+# Conda CI/CD environment for GH Actions.
```

Comment:
What is a "CI/CD" environment, and what are "GH Actions"

Reply:
Thanks for this, it is a good idea to be clear and explicit about the intention of code (and anything in life haha).

Updated this file with a clearer description.